contracts/mesh-consumer/Cargo.toml

```diff
@@ -41,3 +41,7 @@ thiserror        = { workspace = true }
 [dev-dependencies]
 mesh-testing  = { workspace = true }
 cw-multi-test = { workspace = true }
+
+[[bin]]
```

Comment:
any link to docs about this? just to learn what it means for myself.

Reply:
nevermind, found it: https://doc.rust-lang.org/cargo/reference/cargo-targets.html#binaries